addons/web/static/tests/views/list_tests.js

```diff
@@ -399,6 +402,37 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('editable list view: no active element', function (assert) {
+        assert.expect(1);
```

Comment:
Add you custom data in your tests as follow:

```
this.data['bar'] = {
   fields: {
      // put field definitions here
   },
   records: [
      // put records here
   ]
};
```

Reply:
Also, I would add more assert in this test, because it may easily pass successfully even though there's nothing that has been tested.

In particular, you could check that there a list with at least one record, and check the format of each column of a record.

That way, you have more confidence that the data are correctly configured.